lib/src/mock.dart

```diff
@@ -926,7 +926,10 @@ _InOrderVerification get verifyInOrder {
     throw StateError(_verifyCalls.join());
   }
   _verificationInProgress = true;
-  return <T>(List<T> _) {
+  return <T>(List<T> verifyOrderCalls) {
+    if (verifyOrderCalls.whereType<VerificationResult>().isNotEmpty) {
```

Comment:
I see, it will not work, you mean having a validation like this in `_makeVerify`?
```dart
if (_verificationInProgress) {
    fail('There is already a verification in progress, '
          'check if it was not called with a verify argument(s)');
}
_verificationInProgress = true;
return <T>(T mock) {
...
  ```

Reply:
I see, it will not work.

Your suggestion is to have a validation like this in `_makeVerify`?
```dart
if (_verificationInProgress) {
    fail('There is already a verification in progress, '
          'check if it was not called with a verify argument(s)');
}
_verificationInProgress = true;
return <T>(T mock) {
...
```
I tested here, and it worked :)

Reply:
Excellent. If you can update your PR with this, I can merge it.

Reply:
done ;)